src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/AnalysisEntityFactory.cs

```diff
@@ -451,12 +451,6 @@ private AnalysisEntity GetOrCreateForFlowCapture(CaptureId captureId, ITypeSymbo
                 return false;
             }
 
-            // Workaround for https://github.com/dotnet/roslyn-analyzers/issues/1602
```

Comment:
@Evangelink Can you try to extend the failing unit test mentioned in this issue to run with all available PointsToAnalysisKind option values? Basically the below test should be converted: https://github.com/dotnet/roslyn-analyzers/blob/405247d4063733ef5465361001ad98974d9ba82b/src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DisposeObjectsBeforeLosingScopeTests.cs#L8102
so it has `InlineData` as: https://github.com/dotnet/roslyn-analyzers/blob/405247d4063733ef5465361001ad98974d9ba82b/src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DisposeObjectsBeforeLosingScopeTests.cs#L8703-L8706

Reply:
I think the workaround is still needed for `PointsToAnalysisKind.Complete` non-default option value.

Reply:
It worked for all the cases.